src/resources/pandoc/datadir/_utils.lua

```diff
@@ -1,6 +1,10 @@
 -- _utils.lua
 -- Copyright (C) 2020-2022 Posit Software, PBC
 
+--- The pandoc module. Imported to avoid 'no such variable'
+--- warnings in some editors.
+local pandoc = require 'pandoc'
```

Comment:
We should be careful with `require()` because of https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/pandoc/datadir/init.lua#L1403

I'm not sure I want to add it to this PR...